.config/forest.dic

```diff
@@ -1,21 +1,29 @@
 10
-JWT
-JSON
-tipset/SM
-BLS
-SECP
-Algorand/M
-Merkle
-Filecoin
 APIs
-CBOR
-codec
+Algorand/M
 BLAKE2b
+BLS
+CBOR
+CID
+CIDs
+Filecoin
+IPLD
+JSON
+JWT
+Merkle
+RPC
+SECP
+Secp256k1
```

Comment:
We should consistently use one or the other.

Reply:
So which one do we keep? I feel `SECP` is best in our context.

Reply:
In general, both are used in Filecoin specs, also in a variant of `secpk`.  
https://spec.filecoin.io/